src/inverseRobot.js

```diff
@@ -23,8 +23,19 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const part in robot) {
+    if (inversedRobot.hasOwnProperty(robot[part])) {
+      return null;
+    } else {
+      inversedRobot[robot[part]] = part;
+    }
```

Comment:
you don't need `else` here
```suggestion
    if (inversedRobot.hasOwnProperty(robot[part])) {
      return null;
    }
     
     inversedRobot[robot[part]] = part;
```